Foundation/NSHTTPCookieStorage.swift

```diff
@@ -77,7 +77,7 @@ open class HTTPCookieStorage: NSObject {
             return path + fileName
         }
         //if we were unable to create the desired directory, create the cookie file in the `pwd`
-        return fileName
+        return FileManager.default.currentDirectoryPath + fileName
```

Comment:
Before this fix, the path is considered to be the filename which looks like this: "/.cookies.shared". Instead of creating the cookie file in 'pwd', it tries to create the cookie file in the '/' (root) directory.